test/integ/test_dbapi.py

```diff
@@ -735,3 +735,30 @@ def test_escape(conn_local):
             assert (
                 i == row[0]
             ), f"newline not properly converted, got {row[0]}, should be {i}"
+
+
+@pytest.mark.skipolddriver
+def test_callproc(conn_local):
+    with conn_local() as con:
+        cur = con.cursor()
+        executeDDL1(cur)
+        cur.execute(
+            """
+            create or replace procedure output_message(message varchar)
```

Comment:
Please feel free to update all the tests here to use the name randomizer and temporary tables